knowledge-base/badge-container-no-children.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Add a Badge to a Container that Cannot Have Children
```

Comment:
Titles can skip articles. The other change is for shortness and to use different wording in the title and page title

```suggestion
title: Add Badge to Container With No Children
```